hack/verify-bazel.sh

```diff
@@ -17,23 +17,33 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-TESTINFRA_ROOT=$(git rev-parse --show-toplevel)
-TMP_GOPATH=$(mktemp -d)
-cd "${TESTINFRA_ROOT}"
+cd $(git rev-parse --show-toplevel)
 
-OUTPUT_GOBIN="${TESTINFRA_ROOT}/_output/bin"
-GOBIN="${OUTPUT_GOBIN}" go install ./vendor/github.com/bazelbuild/bazel-gazelle/cmd/gazelle
-GOBIN="${OUTPUT_GOBIN}" go install ./vendor/github.com/kubernetes/repo-infra/kazel
+deprecated-verify() {
+  OUTPUT_GOBIN="${TESTINFRA_ROOT}/_output/bin"
```

Comment:
`TESTINFRA_ROOT` was removed above, I'd probably keep this as is and put `TESTINFRA_ROOT` back. the deprecated version will be pretty quick on recent go this way.

Reply:
Done